velox/exec/Task.h

```diff
@@ -703,7 +707,8 @@ class Task : public std::enable_shared_from_this<Task> {
       std::shared_ptr<core::QueryCtx> queryCtx,
       ExecutionMode mode,
       ConsumerSupplier consumerSupplier,
-      std::function<void(std::exception_ptr)> onError = nullptr);
+      std::function<void(std::exception_ptr)> onError = nullptr,
+      int32_t priority = 0);
```

Comment:
Can we move this ahead of onError? We could use macro for backward compatibility?
Shall we call this memoryArbitrationPriority (or memoryReclaimPriority) to be specific? Thanks!